flow/result.go

```diff
@@ -123,15 +123,15 @@ func GetBlockEvents(host string, height uint64, eventType string) {
 
 func printEvents(events []flow.Event, txID bool) {
 	if len(events) == 0 {
-		fmt.Println("  None")
+		fmt.Println(Indent + "None")
 	}
 	// Basic event info printing
 	for _, event := range events {
-		fmt.Printf("  Event %d: %s\n", event.EventIndex, event.String())
+		fmt.Printf(Indent+"Event %d: %s\n", event.EventIndex, event.String())
 		if txID {
-			fmt.Printf("  Tx ID: %s\n", event.TransactionID)
+			fmt.Printf(Indent+"Tx ID: %s\n", event.TransactionID)
 		}
-		fmt.Println("    Fields:")
+		fmt.Println(Indent + Indent + "Fields:")
```

Comment:
There Is inconsistent formatting in this file.

Reply:
I'm not sure what inconsistencies you mean. If you're talking about the spacing between the `+` and the variables/strings, that's just what `gofmt` forces.

I can rejig the printing logic abit to get around this, if it looks too odd to you